src/app/change-account-selector/change-account-selector.component.html

```diff
@@ -22,35 +26,40 @@
     <i class="fas fa-angle-down text-grey5 change-account-selector__down-arrow"></i>
   </div>
 
+</div>
   <!-- Drop-down (open state) -->
-  <div *ngIf="selectorOpen">
+  
+      <!-- Added Class -->
+  <div *ngIf="selectorOpen" class="fs-15px fc-default px-5px cursor-pointer change-account-selector_list br-4px change-account-selector_list__hover">
     <div *ngFor="let user of globalVars.userList; let rowNum = index">
       <div
         *ngIf="!globalVars.loggedInUser || user.PublicKeyBase58Check != globalVars.loggedInUser.PublicKeyBase58Check"
         (mouseover)="hoverRow = rowNum"
-        class="pt-5px pb-5px d-flex align-items-center change-account-selector__hover"
+        class="pt-10px pb-10px pl-10px pr-10px change-account-selector_list__inner d-flex align-items-center"
       >
         <div
           (click)="_switchToUser(user)"
-          class="pl-10px pr-10px change-account-selector__ellipsis-restriction"
+          class="align-items-center change-account-selector__ellipsis-restriction"
           style="flex-grow: 1"
         >
-          {{ user.ProfileEntryResponse?.Username ? user.ProfileEntryResponse.Username : user.PublicKeyBase58Check }}
+        
+      <!-- Added User Image in List-->
+        <div class="feed-create-post__avatar d-inline-block change-account-selector__acount-image" [avatar]="user.PublicKeyBase58Check"></div>
+        <div class="d-inline-block change-account-selector__acount-name">{{ user.ProfileEntryResponse?.Username ? user.ProfileEntryResponse.Username : user.PublicKeyBase58Check }}</div>
```

Comment:
something here is no longer working for long names:
![image](https://user-images.githubusercontent.com/81658138/124683268-8648d680-de81-11eb-9a37-061e088f872b.png)

Reply:
I will check why long name is hidden